diff --git a/src/nnue/features/half_kp.cpp b/src/nnue/features/half_kp.cpp
index b52a45f252..be9c93d640 100644
--- a/src/nnue/features/half_kp.cpp
+++ b/src/nnue/features/half_kp.cpp
@@ -23,9 +23,9 @@
 
 namespace Eval::NNUE::Features {
 
-  // Orient a square according to perspective (rotates by 180 for black)
+  // Orient a square according to perspective (flip rank for black)
   inline Square orient(Color perspective, Square s) {
-    return Square(int(s) ^ (bool(perspective) * 63));
+    return Square(int(s) ^ (bool(perspective) * SQ_A8));
   }
 
   // Index of a feature for a given king position and another piece on some square
